index.html

```diff
@@ -13,10 +13,70 @@
+        </div>
+        <div >
+          <h3 id="contact"> contact:</h3>
+          <div class="card1">
+          <p><a href="marzi.aghchay@gmail.com"><i class="fa fa-envelope" ></i></a></p>
+          <p><a href="https://www.linkedin.com/in/marziyeh-azhdari-3685b0246/"><i class="fa fa-linkedin-square" ></i></a></p>
+          <p><a href="https://en.wikipedia.org/wiki/Iran"><i class="fa fa-wikipedia-w"></i></a></p>
+          
+          
+        </div>
```

Comment:
Here again, is important to use indentation correctly. It is unclear if this div closes the one in line 64 or in line 62 based on indentation.